index.js

```diff
@@ -75,6 +76,14 @@ module.exports = function (target, dest, opts) {
                 fetchArgs.push('--no-save');
             }
 
+            if (opts.save_exact) {
```

Comment:
This if-else (lines 79-85) needs to be combined with the if-else right above it (lines 71-77)